acm/odh-core/acm-observability/kustomization.yaml

```diff
@@ -14,6 +14,14 @@ configMapGenerator:
         observability.open-cluster-management.io/dashboard-folder: "Edge"
       labels:
         grafana-custom-dashboard: "true"
+  - name: grafana-dashboard-edge-inference-metrics-housing-and-binking
```

Comment:
```suggestion
  - name: grafana-dashboard-edge-inference-metrics-housing-and-biking
```

Reply:
Also, apparently this dashboard should either amend or replace the existing one, but this appears to be adding a second dashboard while leaving the first one as it is.

Reply:
Yes, because the old dashboard was broke for a while the plan was to create a separate one and then add it back when it was fixed